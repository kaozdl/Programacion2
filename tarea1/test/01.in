
nada El string 'nada' se convertirÃ¡ en el natural 0.
