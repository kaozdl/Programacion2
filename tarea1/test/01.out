La primera línea está en blanco. 
La conversión es 0.
El inicio numérico de la palabra leida era 0.
